Dockerfile-admiral

```diff
@@ -25,7 +25,7 @@ RUN pip3 install --upgrade pip
 RUN apk add gcc musl-dev python3-dev libffi-dev openssl-dev make && \
     pip3 install cryptography
 
-RUN if [ -n "${INSTALL_IPYTHON}" ]; then pip install ipython pytest; fi
+RUN if [ -n "${INSTALL_IPYTHON}" ]; then pip install ipython pytest mongomock; fi
```

Comment:
I don't think this belongs here.  Note that `mongomock` [is already included as part of the `test` extra listed in `setup.py`](https://github.com/cisagov/admiral/blob/88e530496c4a06b177a58f6927281420a8c7b801/setup.py#L87).

If you are attempting to run the tests from inside the Docker container then you should instead temporarily modify _your local copy_ of `Dockerfile-admiral` so that line 34 becomes:
```python
pip install -e .[test]
```

This change should not be committed to the repository, though.

Reply:
@felddy, can you explain why `pytest` is listed in the line above?  I don't see it as necessary for the purposes of running the admiral either.  `pytest` is included as part of the `test` extra, just like `mongomock`.

Reply:
It looks like it's using the `INSTALL_IPYTHON` environment variable to set up a development Docker image per https://github.com/cisagov/admiral/commit/147f9a8dbf9421f83efb7c7034ed2250da786d8f.

Reply:
OK...shouldn't it instead just `pip install -e .[test]` or (`pip install -e .[dev]`) vice `pip install -e .` depending on the value of `INSTALL_IPYTHON`?  That seems simpler that re-listing all the dev dependencies in the `Dockerfile`.

Reply:
That's probably a saner approach (preferring `--editable` over `-e`).

Reply:
Please see commit 4cb31d8.